concent_api/core/subtask_helpers.py

```diff
@@ -75,6 +75,25 @@ def update_timed_out_subtasks(
                 queue               = PendingResponse.Queue.ReceiveOutOfBand,
                 subtask             = subtask,
             )
+        elif subtask.state == Subtask.SubtaskState.VERIFICATION_FILE_TRANSFER.name:  # pylint: disable=no-member
+            locked_subtask = Subtask.objects.select_for_update().get(pk=subtask.pk)
+
+            update_subtask_state(
```

Comment:
Just for myself, we are not making payment here because it is providers responsibility to upload the files on time ?